pyslurm/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "20.02.5.0"
+__version__ = "20.11.2.0"
```

Comment:
Could we change this to `20.11.8.0` to match the version of Slurm this was compiled against?

Reply:
sure, done!